GetMessages/getMessagesFunctions/getMessages.fallback.ts

```diff
@@ -118,9 +110,7 @@ export const enrichContentData = (
   context: Context,
   messageModel: MessageModel,
   blobService: BlobService,
-  redisClient: RedisClient,
-  remoteContentConfigurationModel: RemoteContentConfigurationModel,
-  remoteContentConfigurationTtl: NonNegativeInteger,
+  rcConfigurationUtility: RCConfigurationUtility,
   categoryFetcher: ThirdPartyDataWithCategoryFetcher
   // eslint-disable-next-line max-params
```

Comment:
maybe we can remove this disable now

Reply:
done in https://github.com/pagopa/io-functions-app-messages/pull/51/commits/c539e77784d456bf922ab57be29284a8e67f4ba7